tests/scalers/elasticsearch/elasticsearch_test.go

```diff
@@ -355,6 +415,7 @@ func addElements(t *testing.T, count int) {
 		_, err = ExecuteCommand(fmt.Sprintf("%s -XPOST http://localhost:9200/%s/_doc -d '%s'", kubectlElasticExecCmd, indexName, result))
 		assert.NoErrorf(t, err, "cannot execute command - %s", err)
 	}
+	time.Sleep(10 * time.Second)
```

Comment:
I realize the test codebase is full of these sleeps, but we would like to put a little more emphasis on making the tests faster and more deterministic. Can you please help me better understand this addition? Perhaps together we can figure out how to avoid it or replace it with a different type of condition, even polling would suffice.

Reply:
Good feedback. I added this line when I was testing the e2e. I don't think it is needed anymore. I will remove it.

Reply:
I have removed the `time.Sleep`